app/src/main/java/com/hemdan/mvvm/ui/main/actorslist/ActorsAdapter.kt

```diff
@@ -0,0 +1,67 @@
+package com.hemdan.mvvm.ui.main.actorslist
+
+import android.content.Context
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.hemdan.mvvm.R
+import com.hemdan.mvvm.data.model.PopularInfo
+import java.util.ArrayList
+
+class ActorsAdapter(private val info: ArrayList<PopularInfo>, private var context: Context)
```

Comment:
@MonaAliMahmoud don't pass the items and context in the constructor if you need context you can access it in the adapter view.context for example if you need it for image loading and for items it's recommended to create public method inside adapter called insertitems() for example and has notifydatasetchanged called after adding items to list